src/console/Normalize/Processor.cs

```diff
@@ -90,5 +109,49 @@ public async Task ConsumeAsync(IEnumerable<IEventMessage> events)
                     eventStats.TotalEventsProcessedBytes);
             }
         }
+
+        private static AsyncPolicy CreateRetryPolicy(ITelemetryLogger logger)
+        {
+            bool ExceptionRetryableFilter(Exception ee)
+            {
+                switch (ee)
+                {
+                    case AggregateException ae when ae.InnerExceptions.Any(ExceptionRetryableFilter):
+                    case OperationCanceledException _:
+                    case HttpRequestException _:
+                    case EventHubsException _:
+                    case AuthenticationFailedException _:
+                    case RequestFailedException _:
```

Comment:
Since we are calling the Azure.Messaging.EventHubs.Producer.EventHubProducerClient downstream, I believe the Producer will throw an Azure.Messaging.EventHubs.EventHubsException instead of a Microsoft.Azure.EventHubs.EventHubsException (legacy).

I think this should be updated to reference the new SDK exception (or both). Feel free to double check. I testing by changing my Normalization Event Hub to be something bogus in order to log an error

Reply:
Thanks for pointing that out. Does this entire class need to updated to use the new API?

Reply:
Actually I see that other interfaces we created use the old API. Its a bit confusing... are their plans to update the project to use a single API?

Reply:
It would be good to transition to the new one, but that should probably be its own work item. It may be harder than just doing a find and replace.

Reply:
Question - When an EventHubsException occurs would we expect `ExceptionRetryableFilter` to return `true` and retry?

For some reason for me when I intentionally log an EventHubsException the code seems to still hit the default block in that switch statement and will return false. I am wondering if it is because it is a `System.Exception {Azure.Messaging.EventHubs.EventHubsException}` or something else with the switch statement that is going on, or if I am misunderstanding how `ExceptionRetryableFilter` works. I was thinking `ExceptionRetryableFilter` would hit the block of code at the at the end that returns true so that it retries.

Reply:
Nevermind - it works fine when pulling in the latest changes.